test/BoardTests.cpp

```diff
@@ -70,10 +70,147 @@ TEST_F(BoardTests, checkBoardArgumentBoard_WithGetBoardLinesFunction)
     Lines vec2 = {vec, vec};
 
     Board board(3, 3, vec2, vec2);
-    ASSERT_EQ(board.getBoardLines().at(1)[1],BoardFields::UNKNOWN);
-    ASSERT_EQ(board.getBoardLines().at(1)[0],BoardFields::UNKNOWN);
-    ASSERT_EQ(board.getBoardLines().at(0)[1],BoardFields::UNKNOWN);
-    ASSERT_EQ(board.getBoardLines().at(0)[0],BoardFields::UNKNOWN);
+    ASSERT_EQ(board.getBoardLines().at(1)[1], BoardFields::UNKNOWN);
+    ASSERT_EQ(board.getBoardLines().at(1)[0], BoardFields::UNKNOWN);
+    ASSERT_EQ(board.getBoardLines().at(0)[1], BoardFields::UNKNOWN);
+    ASSERT_EQ(board.getBoardLines().at(0)[0], BoardFields::UNKNOWN);
+}
+
+TEST_F(BoardTests, checkBoardSetField)
+{
+    Board board(3, 2, {}, {});
```

Comment:
Please do not overcompilcate test scenarios. They should be as simple as possible. If you can create 1x1 Board then it can be used, if not then 2x2 is ok.

Reply:
Testing symmetrical/square Boards makes too big opportunity to have false success.